### ISSUE TITLE
No setting to limit how many Contributions we see

### ISSUE THREAD

**Issue by jidanni** [commons-app/apps-android-commons#3970]
In settings there is a Recent Upload Limit setting.
Well, there should also be a Contributions Limit setting.
Or maybe they are the same thing.
1. Install the app
2. Login

On the Home screen, on the left, there are our thousands of contributions, covering many years.
We think, what a waste, we want to save bandwidth.
Why can't there just be a setting to show 10 of them. Sure we tried #3969.

**Comment by jidanni** [comment]
What's worse is apparently upon launching the app, each time, all the images are downloaded from the network again!

**Comment by sivaraam** [comment]
> What's worse is apparently upon launching the app, each time, all the images are downloaded from the network again!

Yeah, that's a current limitation of the app. A "limited connection mode" feature is being implemented now (cf. #746). This might be one thing that we can address in that feature.

Also, you might be interested in chiming in there :)
